APIcall/data_collection.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+import requests
+import argparse
+from datetime import date
+
+def api_311(start_date, end_date, skip, limit):

```

Comment:
Please add a docstring. See details here: https://google.github.io/styleguide/pyguide.html#383-functions-and-methods